File: NodeJS/scraping/wait_for_element.js
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage();
  await page.goto('https://example.com');
  await page.waitForSelector('h1');
  const text = await page.textContent('h1');
  console.log('Found:', text);
  await browser.close();
})();

File: Python/scraping/wait_for_element.py
from playwright.sync_api import sync_playwright

with sync_playwright() as p:
    browser = p.chromium.launch()
    page = browser.new_page()
    page.goto("https://example.com")
    page.wait_for_selector("h1")
    print("Found:", page.text_content("h1"))
    browser.close()
